packages/riipen-ui/src/components/Form.jsx

```diff
@@ -163,82 +116,117 @@ class Form extends React.Component {
     return renderObject(errors);
   };
 
-  renderErrorNode = error => error;
-  renderErrorString = error => <Typography>{error}</Typography>;
-
-  render() {
-    const { children, classes, error, errors, ...other } = this.props;
-
-    const linkedStyles = this.getLinkedStyles();
-
-    const className = clsx(classes);
+  const renderErrorNode = val => val;
+  const renderErrorString = val => <Typography>{val}</Typography>;
 
-    // Remove any unwanted props from "other"
-    delete other.enter;
-    delete other.errorScroll;
+  const linkedStyles = getLinkedStyles();
 
-    let errorRenderer;
-    let errorsRenderer;
+  let errorRenderer;
+  let errorsRenderer;
 
-    if (error) {
-      if (typeof error === "string") {
-        errorRenderer = this.renderErrorString;
-      } else if (React.isValidElement(error)) {
-        errorRenderer = this.renderErrorNode;
-      }
+  if (error) {
+    if (typeof error === "string") {
+      errorRenderer = renderErrorString;
+    } else if (React.isValidElement(error)) {
+      errorRenderer = renderErrorNode;
     }
+  }
 
-    if (errors) {
-      if (
-        Array.isArray(errors) &&
-        errors.length > 0 &&
-        typeof errors[0] === "string"
-      ) {
-        errorsRenderer = this.renderErrorsString;
-      } else if (
-        Array.isArray(errors) &&
-        errors.length > 0 &&
-        React.isValidElement(errors[0])
-      ) {
-        errorsRenderer = this.renderErrorsNode;
-      } else if (typeof errors === "object" && Object.keys(errors).length > 0) {
-        errorsRenderer = this.renderErrorsObject;
-      }
+  if (!isEmpty(errors)) {
+    if (
+      Array.isArray(errors) &&
+      errors.length > 0 &&
+      typeof errors[0] === "string"
+    ) {
+      errorsRenderer = renderErrorsString;
+    } else if (
+      Array.isArray(errors) &&
+      errors.length > 0 &&
+      React.isValidElement(errors[0])
+    ) {
+      errorsRenderer = renderErrorsNode;
```

Comment:
can remove errors.length>0 since `!Object.keys(['i have length']).length === false`, so it's checked already by `!isEmpty`
```suggestion
    if (
      Array.isArray(errors)
    ) {
      if(typeof errors[0] === "string"){
            errorsRenderer = renderErrorsString;
      } else if(React.isValidElement(errors[0])){
            errorsRenderer = renderErrorsNode;      
      }
```